pkg/ibmvpc/examples/out/explain_out/GroupingExternalSG1_all_vpcs_explain.txt

```diff
@@ -1,7 +1,7 @@
 Explaining connectivity from vsi1-ky to 161.26.0.0/8 within test-vpc1-ky
 ========================================================================
 
-Allowed connections from vsi1-ky[10.240.10.4] to Public Internet 161.26.0.0/16: protocol: UDP
+Connections from vsi1-ky[10.240.10.4] to Public Internet 161.26.0.0/16: protocol: UDP
```

Comment:
Does the new sentence make sense? That allows connections from src to dst are UDP? 
I think not (it was clear before)

Reply:
I think it is clear enough.. the term "allows" is not general enough, for LB it's not a matter of "allowing"